src/backend/base/langflow/components/langchain_utilities/SQLDatabase.py

```diff
@@ -13,11 +14,7 @@ def build_config(self):
             "uri": {"display_name": "URI", "info": "URI to the database."},
         }
 
-    def clean_up_uri(self, uri: str) -> str:
-        if uri.startswith("postgresql://"):
-            uri = uri.replace("postgresql://", "postgres://")
```

Comment:
Why remove this? If the user passes the incorrect url they'll open an issue in vain

Reply:
You're right. I forgot that part, fixed